src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -56,12 +56,13 @@ public PlayerJoinEvent send(PlayerJoinEvent message) throws Exception {
         newPlayer.setPlayerUsername(message.getPlayerName());
         newPlayer.setHost(false);
         gameService.joinGame(message.getGameId(), newPlayer);
-        messagingTemplate.convertAndSend("/game1/messages/" + message.getGameId(),message);
+        messagingTemplate.convertAndSend("/game1/messages/" + message.getGameId(), message);
```

Comment:
added space after argument within parameters for best practices and readability.
This is recommended by the "Spacing & line breaks" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3